src/main/resources/inspectionDescriptions/RsUselessMatchArm.html

```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+Checks that <code>match</code> expression doesn't have useless (unreachable) arms.
```

Comment:
Note, it also detects unreachable patterns if arm uses or_patterns. So let's reword it to `Checks that <code>match</code> expression doesn't have useless (unreachable) patterns.`